database_creation.py

```diff
@@ -1,11 +1,17 @@
 import mariadb
 import sys
 
+# run this command in linux to start the docker container with maria db
+# docker run -d -p 3306:3306 -e MYSQL_DATABASE=example -e MYSQL_ROOT_PASSWORD=password -e TZ=America/Los_Angeles --name mdb103 mariadb:10.3
```

Comment:
This comment is not needed because the information is included on documentation in the project's ReadMe.